mergekit/architecture.py

```diff
@@ -353,6 +404,7 @@ def _load_all_architectures() -> (
 JSON_ARCHITECTURES, NAME_TO_ARCH = _load_all_architectures()
 MISTRAL_INFO = _load_json_arch("mistral.json")
 QWEN2_INFO = _load_json_arch("qwen2.json")
+DEEPSEEKV2_INFO = _load_json_arch("deepseekv2.json")
 
 
 def get_architecture_info(config: PretrainedConfig) -> ArchitectureInfo:
```

Comment:
Unless I am mistaken which I could be, there needs to be a clause within the `get_architecture_info` function similar to Mixtral's otherwise the code https://github.com/arcee-ai/mergekit/blob/main/mergekit/merge.py#L51 will just pull in just the info associated with the template